collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py

```diff
@@ -127,9 +127,7 @@ def gpu_charts(gpu):
         },
         USER_NUM: {
             'options': [None, 'Number of User on GPU', 'Nr', fam, 'nvidia_smi.user_num', 'stacked'],
-            'lines': [
-                ['nr', 'nr'],
-            ]
+            'lines': []
```

Comment:
should be `['user_num', 'users']`

units  `Nr` => `num`

Am i missing something?

Reply:
I am missing something ...